internal/logging/zerolog.go

```diff
@@ -124,7 +124,10 @@ func InitializeLogger() (zerolog.Logger, func()) {
 			fn()
 		}
 	}
-	return decorate(zerolog.New(io.MultiWriter(writers...))), closeFn
+	logger := decorate(zerolog.New(io.MultiWriter(writers...)))
+	log.Logger = logger
+	zerolog.DefaultContextLogger = &logger
+	return logger, closeFn
 }
```

Comment:
InitializeLogger now initializes both log.Logger and also default loger which `Ctx` function returns when nothing is in the context.